vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java

```diff
@@ -815,7 +820,8 @@ else if (drawSingleDeprecated) {
       SHOW_MOVE_SELECTED,
       SHOW_NON_MOVABLE,
       UNROTATE_PIECES,
-      SHOW_DECK
+      SHOW_DECK,
+      SHOW_OVERLAP,
```

Comment:
Trailing comma.

Reply:
This is legitimate Java code and makes further work on this code easier by allowing to copy+paste the last line and to adjust it according to needs.

Reply:
Yeah I actually "found these" with trailing commas in them and simply extended the pattern.

Reply:
I recommend we leave the commas, but let me know if you want me to take them out.

Otherwise I addressed your other thing and it's perhaps ready to go.